Formula/nerdctl.rb

```diff
@@ -0,0 +1,22 @@
+class Nerdctl < Formula
+  desc "ContaiNERD CTL - Docker-compatible CLI for containerd"
+  homepage "https://github.com/containerd/nerdctl"
+  url "https://github.com/containerd/nerdctl/archive/refs/tags/v0.15.0.tar.gz"
+  sha256 "7caf808a271d47ff069875cd49ceed02cf3fd3c0c334df2d89b3a031b0e6d041"
+  license "Apache-2.0"
+  head "https://github.com/containerd/nerdctl.git", branch: "master"
+
+  depends_on "go" => :build
+  depends_on :linux
+
+  def install
+    ldflags = "-s -w -X github.com/containerd/nerdctl/pkg/version.Version=#{version}"
+    system "go", "build", *std_go_args(ldflags: ldflags), "./cmd/nerdctl"
+  end
+
+  test do
+    output = shell_output("XDG_RUNTIME_DIR=/dev/null #{bin}/nerdctl images 2>&1", 1).strip
```

Comment:
Since we're setting it ourselves, let's do this too.
```suggestion
  test do
    assert_match version.to_s, shell_output("#{bin}/nerdctl --version")
    output = shell_output("XDG_RUNTIME_DIR=/dev/null #{bin}/nerdctl images 2>&1", 1).strip
```

Reply:
Thanks I applied it